app/controllers/users_controller.rb

```diff
@@ -56,20 +56,24 @@ def dpo; end
   def key_contact; end
 
   def edit
-    redirect_to user_path(@user) unless @user.active?
+    redirect_to user_path(@user) unless @user.active? || current_user.support?
   end
 
   def update
     validate_attributes
-    if @user.errors.empty? && @user.update(user_params)
+    if @user.errors.empty? && @user.update(user_params_without_org)
       if @user == current_user
         bypass_sign_in @user
         flash[:notice] = I18n.t("devise.passwords.updated") if user_params.key?("password")
-        if user_params.key?("email")
+        if user_params.key?("email") && user_params[:email] != @user.email
           flash[:notice] = I18n.t("devise.email.updated", email: @user.unconfirmed_email)
         end
 
-        redirect_to account_path
+        if updating_organisation?
```

Comment:
Hmm, I don't think it was explicit whether they should or shouldn't be able to move themselves in the ACs, but it might be useful on staging if people can move themselves around as we had to manually move people for testing purposes in the past. Let me know if there are any concerns with this tho